README.md

```diff
@@ -65,13 +65,13 @@ Finally, if running PHP in a Web server, restart the Web server.
 
 ## Install (UNIX)
 
-For full instructions on installing the drivers on all supported Unix platforms, see [the installation instructions on Microsoft Docs](https://docs.microsoft.com/sql/connect/php/installation-tutorial-linux-mac).
+For full instructions on installing the drivers on all supported Unix platforms, see [the installation instructions on Microsoft Docs][odbcinstructions].
```

Comment:
odbcinstructions links to the ODBC install instructions - we want to link to the PHP install instructions don't we?